pkg/apis/softwarecomposition/v1beta1/generated.proto

```
@@ -131,6 +131,51 @@ message CallStackNode {
+  // Threshold is the maximum number of unique children allowed at any
+  // trie node under Prefix before that node collapses to a single
+  // dynamic identifier.
+  optional int32 threshold = 2;
+}
+
+// CollapseConfiguration is a cluster-scoped resource carrying per-prefix
+// thresholds for the dynamic-path-detector's open/endpoint collapse step.
+// The storage server's deflate path reads the singleton (name "default")
+// and feeds its entries into NewPathAnalyzerWithConfigs at runtime.
+message CollapseConfiguration {
+  optional .k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
```

Buf (1.69.0)

[error] cannot find `.k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta` in this scope

(COMPILE)